src/components/claims/claim-details/components/claim-items/components/ItemForm.tsx

```
@@ -26,40 +32,59 @@ export const ItemForm: React.FC<{
   const [dateOfPurchase, setDateOfPurchase] = React.useState('')
   const [note, setNote] = React.useState('')
   const [purchasePriceCurrency, setPurchasePriceCurrency] = React.useState('')
-
+  const [autoValuation, setAutoValuation] = React.useState<string | null>('')
+  const [customValuation, setCustomValuation] = React.useState('')
   const [contractMarketInfo] = useContractMarketInfo(memberId ?? '')
-  const validPurchasePrice = /^[0-9]*$/g.test(purchasePrice)
   const defaultCurrency = contractMarketInfo?.preferredCurrency ?? 'SEK'
 
   const [
     upsertClaimItem,
     { loading: loadingUpsertClaimItem },
   ] = useUpsertClaimItemMutation()
 
-  const addNewClaimItemRequest: UpsertClaimItemInput = {
+  const formData: UpsertClaimItemInput = {
     claimId,
     itemFamilyId: selectedItemCategories[0]?.id ?? null,
     itemTypeId: selectedItemCategories[1]?.id ?? null,
     itemBrandId: selectedItemCategories[2]?.id ?? null,
     itemModelId: selectedItemCategories[3]?.id ?? null,
-    dateOfPurchase: dateOfPurchase === '' ? null : dateOfPurchase,
-    purchasePriceAmount: purchasePrice !== '' ? Number(purchasePrice) : null,
-    purchasePriceCurrency: purchasePrice !== '' ? purchasePriceCurrency : null,
+    dateOfPurchase:
+      dateOfPurchase !== '' && dateOfPurchase.length === 10
+        ? dateOfPurchase
+        : null,
+    purchasePriceAmount: isNotEmpty(purchasePrice)
+      ? Number(purchasePrice)
+      : null,
+    purchasePriceCurrency: isNotEmpty(purchasePrice)
+      ? purchasePriceCurrency
+      : null,
+    valuationAmount: Number(autoValuation) ?? null,
+    customValuationAmount:
+      isNotEmpty(customValuation) && isValidNumber(customValuation)
```

Similarly, the form itself should still send the `customValuation` down even though it is invalid for the backend to break. Otherwise we send down nothing even though they made a type for example `1000b` we send down `null` which is weird

---

The form itself should validate it of course, maybe disable the button and make the correct input red or something. But the send to the backend should not ignore the error.